bindings/python/tokenizers/normalizers/__init__.pyi

```diff
@@ -7,7 +7,7 @@ class Normalizer:
     Normalizer will return an instance of this class when instantiated.
     """
 
-class BertNormalizer:
+class BertNormalizer(Normalizer):
```

Comment:
Is there a specific reason you added these? Or is it just for accuracy?
I think we should add these for all the other parts too if we keep it here (PreTokenizer, Decoder, ...).

Reply:
It's more syntactic sugar as `unicode_normalizer_from_str() -> Normalizer` returns a "Normalizer" and I wanted to reflect the relationship between the Normalizers we have.

We can also stay the way it is currently (i.e. no inheritance) and I change the prototype of the function to `unicode_normalizer_from_str() -> Union[NFC, NFD, NFKC, NFKD]`

What do you prefer @n1t0 ?

Reply:
I think the way you did it is the most accurate. Everywhere we expect `Normalizer` so let’s keep it this way. 
I would just like to keep everything consistent and add it for other classes too. You can do it here if you want, otherwise I’ll do it separately!